sklearn/neural_network/base.py

```diff
@@ -0,0 +1,228 @@
+"""Utilities for the neural network modules
```

Comment:
I would make this file private by starting it with an underscore. I'm not sure we want to add that many public function that are all one-liners.